.gitignore

```diff
@@ -17,4 +17,7 @@ node_modules/
 yalc.lock
 
 # Databases
-database*
\ No newline at end of file
+database*
+
+# Default config
+config.json
```

Comment:
sorry, it was me who suggested. but maybe we can make the pattern:

```
/config*.json
````

As we can have all sort of config files. 



<img width="410" alt="image" src="https://github.com/cowprotocol/watch-tower/assets/2352112/058b8907-5a95-48af-a8cd-ed71598ccc0b">


Also, i made it relative to the root, so this don't byte us if we create a config somewhere in the `src`

Reply:
In general, I'd be a fan of only ignoring the recommended / assumed default configuration (which is also that which we document), however it's low hanging fruit to ignore all configs. This has been included :+1: